src/containers/shared/components/test/DomainLink.test.tsx

```diff
@@ -38,4 +38,13 @@ describe('DomainLink', () => {
     expect(wrapper.find('a').props().href).toEqual('https://sologenic.com')
     wrapper.unmount()
   })
+  it('handles domain link with domain provided in HEX-encoded format', () => {
+    const wrapper = mount(
+      <DomainLink decode domain="68747470733A2F2F6578616D706C652E636F6D" />,
+    )
+    expect(wrapper.find('a').props().className).toEqual('domain')
+    expect(wrapper.find('a').text()).toEqual('https://example.com')
+    expect(wrapper.find('a').props().href).toEqual('https://example.com')
+    wrapper.unmount()
+  })
```

Comment:
This is a bit more understandable:
```suggestion
    const url = 'https://example.com'
    const urlHex = encodeHex(url)
    const wrapper = mount(
      <DomainLink decode domain={urlHex} />,
    )
    expect(wrapper.find('a').props().className).toEqual('domain')
    expect(wrapper.find('a').text()).toEqual(url)
    expect(wrapper.find('a').props().href).toEqual(url)
    wrapper.unmount()
  })
```

Reply:
As for the additional test that is created in the _DomainLink.test.tsx_, I just followed the format of the previous tests in the same test file. If only this test is changed, the file will be non-uniform. If you still want to change it the way you mentioned above, let me know.

Reply:
Changed it for better readability: https://github.com/ripple/explorer/pull/856/commits/b8fc786640d384fc3d893dca5b4410f6f4ad74a6